### PR TITLE
Add AntDesign to icons in src/helps/getIconType.js

### PR DESCRIPTION
**Pull request by osdnk**
Base on: https://github.com/react-native-training/react-native-elements/pull/1474
## Motivation
Wish to use new icon set from RNVI

## Changes
Add proper import and bump RNVI version

### CODE REVIEW

**Comment by osdnk** [comment]
cc @d-p-s @iRoachie

**Comment by iRoachie** [comment]
Also add it to the list of icons here https://github.com/react-native-training/react-native-elements/blob/next/docs/icon.md#available-icon-sets

**Comment by osdnk** [comment]
Ok, I'll do it soon

**Comment by osdnk** [comment]
@iRoachie 
Please take a look

File: docs/icon.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The current list of available icons sets are:
 * [zocial](http://weloveiconfonts.com/)
 * [entypo](http://www.entypo.com/)
 * [feather](https://feathericons.com/)
+* [AntDesign](http://beta.ant.design/components/icon/)
```
Comment (iRoachie) [inline docs/icon.md]: Use lowercase here, it has the match the value in getIconType.

```suggestion
* [antdesign](http://beta.ant.design/components/icon/)
```

**Comment by osdnk** [comment]
Done

**Comment by iRoachie** [comment]
Lastly could you do a rebase from the `next` branch? Seems you have some outdated commits.

**Comment by osdnk** [comment]
I could do it tommorow. Wouldn't you like simply squash my commits?

**Comment by iRoachie** [comment]
Nvmd it's fine.

**Comment by iRoachie** [comment]
Thanks for pulling this off 💯 Great job

**Comment by iRoachie** [comment]
We may actually have to delay this a bit, AntDesign isn't available through expo, and it throws an error.

**Comment by osdnk** [comment]
@iRoachie 
I have contacted with Expo team and they do confirm that it will be available in a new release (31.0.0), which will be available soon (it's matter of few days)
CC: @tsapeta

**Comment by iRoachie** [comment]
Thanks!

**Comment by Monte9** [comment]
This fixed it for me: `yarn add @expo/vector-icons`

https://github.com/expo/expo/issues/1083#issuecomment-350198071

**Pull request merged**